common/models/user.js

```diff
@@ -719,7 +727,7 @@ module.exports = function(User) {
     // email validation regex
     var re = /^(([^<>()[\]\\.,;:\s@\"]+(\.[^<>()[\]\\.,;:\s@\"]+)*)|(\".+\"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
 
-    UserModel.validatesFormatOf('email', { with: re, message: 'Must provide a valid email' });
+    UserModel.validatesFormatOf('email', { with: re, message: g.f('Must provide a valid email') });
```

Comment:
Regexp is one of the challenges for globalization.  In short, it does not work because the word order in sentences varies from language to language.  This case requires careful refactoring.